src/ddprof_worker.cc

```diff
@@ -88,9 +88,26 @@ DDRes report_lost_events(DDProfContext &ctx) {
   return {};
 }
 
+inline int64_t get_mono_to_realtime_offset(
```

Comment:
Why `inline` ?

Reply:
Oops, fixed.  I was thinking something else at the time.